http/incidence_client.go

```
@@ -2,15 +2,43 @@ package http
 
 import (
 	"context"
-	"fmt"
-
+	"encoding/json"
 	"github.com/swisscovidtravel/conav"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
 )
 
 var _ conav.IncidenceGetter = &IncidenceClient{}
 
 type IncidenceClient struct{}
 
+var switchServers = []string{"AG", "BS", "SG", "ZH", "JU", "...", "conav.dvob.org"}
+// test with our solothurn server
+var url = "https://" + switchServers[6] + "/incidences/"
+
 func (i *IncidenceClient) GetIncidences(ctx context.Context, filter *conav.IncidenceFilter) ([]*conav.Incidence, error) {
-	return nil, fmt.Errorf("not implemented")
+	var incidences []*conav.Incidence
+
+	// Set up url to call
+	if filter.BFSID != 0 { // if value is set
+		url += strconv.Itoa(filter.BFSID) + "/"
+	}
+	if filter.DateFrom != "" && filter.DateTo != "" { // if values are set
+		url += "?dateFrom=" + filter.DateFrom + "&dateTo=" + filter.DateTo
+	}
+
+	// make the call & save the incidences retrieved
+	response, err := http.Get(url)
+	if err != nil {
+		log.Printf("The HTTP request failed with error %s\n", err)
```

If something fails return the error instead of logging it:
```
return nil, err
```
This way the caller can decide how to handle the error.